packages/libchromaprint/chromaprint.subpackage.sh

```diff
@@ -0,0 +1,6 @@
+TERMUX_SUBPKG_DESCRIPTION="C library for generating audio fingerprints used by AcoustID (fpcalc tool)"
+TERMUX_SUBPKG_DEPENDS="ffmpeg"
+TERMUX_SUBPKG_INCLUDE="
+bin/fpcalc
+share/doc/chromaprint/copyright
```

Comment:
```suggestion
```

No need for this, besides it doesnt work anyway

https://github.com/termux/termux-packages/actions/runs/16001029638/job/45135753010?pr=25227#step:7:18384
```
WARNING: tried to add share/doc/chromaprint/copyright to subpackage 'chromaprint', but could not find it
```